tests/format/typescript/conditional-types/with-balanced-formatting/jsfmt.spec.js

```diff
@@ -2,7 +2,7 @@
 // This test script runs for test files in parent directory,
 // **not** on any files in *this* directory.
 
-const dirpath = `${__dirname}/..`;
+const dirPath = `${__dirname}/..`;
 
 run_spec(dirpath, ["typescript"], {
```

Comment:
missed this line

Reply:
and fixed another line, should be fixed now

Reply:
fixed & rebased with a force-push